meltano.yml

```diff
@@ -25,43 +25,43 @@ plugins:
     - name: linkedin_version
       value: '202207'
     - name: campaign
-      value: ${TAP_LINKEDIN_CAMPAIGN}
+      value: ${TAP_LINKEDIN_ADS_CAMPAIGN}
```

Comment:
This is just an example `meltano.yml` but can we get rid of the default these `value` entries that reference an env var?

Reply:
Removed!